sample-apps/go-sample-app/collection/client.go

```diff
@@ -90,28 +97,22 @@ func StartClient(ctx context.Context) (func(context.Context) error, error) {
 			}
 		}()
 		// pushes any last exports to the receiver
-		err = ctrl.Stop(ctx)
+		err = meterProvider.Shutdown(ctx)
```

Comment:
Should this error be handled and returned?

Reply:
This is now handled